lib/client/api.go

```diff
@@ -1330,6 +1329,20 @@ func NewClient(c *Config) (tc *TeleportClient, err error) {
 	return tc, nil
 }
 
+// ExitStatus returns the exit status of the most recent ssh command.
```

Comment:
```suggestion
// ExitStatus returns the exit status of the most recent SSH command.
```

Is it worth mentioning that it's the caller's responsibility to ensure that the command has already exited before checking this?